### PR TITLE
Update the documentation for web-platform-tests.live

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

**Comment by jugglinmike** [comment]
Rather than make a bunch of inline comments, I've pushed up some commits to implement my suggestions directly. This is a little presumptuous, but most of it is fairly objective. Could you take a look?

File: README.md [hunk 1]
```diff
@@ -87,5 +87,5 @@ To completely white out the Vagrant machine and build a new one from scratch,
 you can use the following command.
 
 ```
-$ vagrant destroy -f && vangrant ssh
+$ vagrant destroy -f && vangrant up
```
Comment (bmac) [inline README.md]: Ooops. I don't know where that `ssh` came from. I use `up` all the time myself.

**Comment by bmac** [comment]
Looks great @jugglinmike.

File: README.md (hunk 1 above)
Comment (jugglinmike) [reply README.md]: This can be your one allowed typo for today

Comment (bmac) [reply README.md]: Oh no. I'm I allowed to take out a loan on tomorrow's typo?

Comment (jugglinmike) [reply README.md]: > Oh no. I'm I allowed to take out a loan on tomorrow's typo?

Yes, but you just spent that, too

**Pull request merged**